packages/block-library/src/social-link/index.php

```diff
@@ -0,0 +1,227 @@
+<?php
+/**
+ * Server-side rendering of the `core/social-link` blocks.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/social-link` block on server.
+ *
+ * @param array $attributes The block attributes.
+ *
+ * @return string Rendered HTML of the referenced block.
+ */
+function render_core_social_link( $attributes ) {
+	$site = ( isset( $attributes['site'] ) ) ? $attributes['site'] : 'Icon';
```

Comment:
I was trying to understand in what situations the `site` att can be null, but I couldn't come up with any. I guess using the `share` icon and adding the `wp-social-link-Icon` is only a protective measure.

Reply:
yeh, just a defensive measure, figure its better to be safe and not assume anything